.changeset/wide-taxes-bathe.md

```
@@ -0,0 +1,5 @@
+---
+'mastra': minor
+---
+
+Added `mastra factory create` to scaffold Factory projects locally with released dependency versions. The command can also clone a custom Git template with `--template`.
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)